src/app.module.ts

```diff
@@ -14,6 +16,10 @@ import { SkillModule } from './skills/skill.module';
     UserModule,
     SkillModule,
     ConfigModule.forRoot(),
+    ServeStaticModule.forRoot({
```

Comment:
This module serves static content from the assets directory at the web root.

So for example to fetch the dist/assets/images/whatever.png image, you'd use http://hostname/images/whatever.png.